src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,17 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform 300ms, color 300ms;
```

Comment:
mentor meant to create a variable for 300ms, because it's used 3 times so would be good to not repeat